packages/lesswrong/lib/sql/tests/testHelpers.ts

```diff
@@ -74,8 +74,29 @@ export const TestCollection2 = {
 
 export const testTable2 = Table.fromCollection(TestCollection2);
 
+export type DbTestObject3 = {
+  _id: string,
+  notNullData: string
+};
+
+export const TestCollection3 = {
+  collectionName: "TestCollection3",
+  _schemaFields: {
+    _id: {
+      type: String,
+    },
+    notNullData: {
+      type: String,
+      nullable: false
+    }
+  }
+} as unknown as CollectionBase<DbTestObject>;
```

Comment:
```suggestion
} as unknown as CollectionBase<DbTestObject3>;
```

Reply:
Whoops, this wasn't working but that was because I'd forgotten the `schemaVersion` field.  Fixed.